src/main/java/io/github/pmckeown/dependencytrack/metrics/MetricsMojo.java

```diff
@@ -10,11 +10,42 @@
 import org.apache.maven.plugin.MojoFailureException;
 import org.apache.maven.plugins.annotations.LifecyclePhase;
 import org.apache.maven.plugins.annotations.Mojo;
+import org.apache.maven.plugins.annotations.Parameter;
 
 import javax.inject.Inject;
 
 /**
- * Provides the capability to print the full set of metrics about a project as determined by the Dependency Track Server
+ * Print the full set of metrics about a project as determined by the Dependency Track Server
+ *
+ * You can optionally define thresholds for failing the build where the number of issues in a particular category
+ * is greater than the threshold you define for that threshold.
```

Comment:
"you define for that threshold" -> "you define for that category"

Reply:
Cheers!